### PR TITLE
Project creation refactor

### PR DESCRIPTION
**Pull request by itailiors**

### CODE REVIEW

File: src/Angor/Client/Pages/View.razor [hunk 1]
```diff
@@ -21,228 +12,237 @@
 @inject IJSRuntime Js;
 @inject ILogger<Browse> Logger;
 
-@inject Angor.Shared.Utilities.NostrConversionHelper NostrHelper
-
+@inject NostrConversionHelper NostrHelper
```
Comment (itailiors) [inline src/Angor/Client/Pages/View.razor:15]: todo, return this to our practice  (my reformat does this..)

File: src/Angor/Client/Pages/Create.razor [hunk 2]
```diff
@@ -83,413 +81,455 @@
...
+            </button>
+            <button type="button" class="btn btn-success" @onclick="CreatProjectTransactionAnCheckPassword" disabled="@creatProjectSpinner">
+                @if (creatProjectSpinner)
+                {
+                    <span class="spinner-border spinner-border-sm" role="status" aria-hidden="true"></span>
+                    <span>Create Project...</span>
+                }
+                else
+                {
+                    <span>Create Project</span>
```
Comment (itailiors) [inline src/Angor/Client/Pages/Create.razor:390]: TODO: Make the button disabled if the nostr and project data wasnt created

File: src/Angor/Client/Pages/Create.razor [hunk 3]
```diff
@@ -756,10 +822,6 @@
             _Logger.LogError(e, e.Message);
             notificationComponent.ShowErrorMessage(e.Message);
         }
-        finally
-        {
-            // nothing to do the spinner will be disabled in the call back
-        }
     }
 
     private async Task CreatProjectTransactionAnCheckPassword()
```
Comment (itailiors) [inline src/Angor/Client/Pages/Create.razor:827]: TODO: Research if there's a better way

File: src/Angor/Client/Pages/View.razor [hunk 4]
```diff
@@ -710,16 +702,16 @@
         input = Regex.Replace(input, @"<style.*?>.*?</style>", string.Empty, RegexOptions.IgnoreCase | RegexOptions.Singleline);
 
         input = Regex.Replace(input, @"<([a-zA-Z][^\s>]*)(\s+[^>]*)?>", match =>
-     {
-     string tag = match.Groups[1].Value;
-     string attributes = match.Groups[2].Value;
+        {
+            var tag = match.Groups[1].Value;
```
Comment (itailiors) [inline src/Angor/Client/Pages/View.razor:706]: check if the reformat change it to be var instead of a string? i have no recollection of changing this

**Comment by itailiors** [comment]
this is ready to review,
but theres a problem that because of the refactor it's annoying to look at (maybe in rider view its better ?)

**Comment by dangershony** [comment]
> Added fix for the utxo bug.  

what was the utxo bug again?

File: src/Angor/Client/Storage/ClientStorage.cs [hunk 5]
```diff
@@ -2,45 +2,41 @@
...
 
 namespace Angor.Client.Storage;
 
 public class ClientStorage : IClientStorage, INetworkStorage
 {
-    private readonly ISyncLocalStorageService _storage;
-    
     private const string CurrencyDisplaySettingKey = "currencyDisplaySetting";
-    
+
```
Comment (dangershony) [inline src/Angor/Client/Storage/ClientStorage.cs:11]: It's very hard to know what has changed in this file, many methods have just moved around.
Can you point me to the changes in this file?

**Pull request closed without merging**